torch/nn/modules/module.py

```diff
@@ -1387,6 +1387,13 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
             key = prefix + name
             if key in state_dict:
                 input_param = state_dict[key]
+                if not torch.is_tensor(input_param):
```

Comment:
Shouldn't this be https://pytorch.org/docs/stable/torch.overrides.html?highlight=is_tensor_like#torch.overrides.is_tensor_like ?

Reply:
This will become more important as we broaden support for custom tensors.